### PR TITLE
README.md updates

### PR DESCRIPTION
**Pull request by UncleGene**
Summary:

- Build instructions for Mac updated to reference non-specific LLVM version
- Added information for Catalina build

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,19 +79,22 @@ Install the required dependencies using either [Homebrew](https://brew.sh/) or
...
-  brew install llvm@7
+  brew install cmake graphviz libpng ninja protobuf wget glog autopep8 llvm
   ```
 
 If using MacPorts, run:
 
   ```bash
-  port install cmake graphviz libpng ninja protobuf-cpp wget llvm-7.0 google-glog
+  port install cmake graphviz libpng ninja protobuf-cpp wget google-glog
+  # Choosse version >=7.
```
Comment (bertmaher) [inline README.md]: typo: s/Choosse/Choose/

File: README.md [hunk 2]
```diff
@@ -104,15 +107,17 @@ on. For a Homebrew-managed installation, run:
...
-  ln -s "/opt/local/libexec/llvm-7.0/bin/clang-tidy" "/usr/local/bin/clang-tidy"
+  ln -s "/opt/local/libexec/llvm-$LLVM_VERSION.0/bin/clang-format" "/usr/local/bin/clang-format"
+  ln -s "/opt/local/libexec/llvm-$LLVM_VERSION.0/bin/clang-tidy" "/usr/local/bin/clang-tidy"
 ```
 
-> **Note:** On newer versions of macOS, Xcode's command line tools come with a
-> non-traditional header layout. In order for Glow to build on newer macOS
-> versions, you might need to install `macOS_SDK_headers_for_macOS_10.14.pkg`
-> manually. For example, on Mojave this package is located in
+> **Note:** Starting macOS Mojave Xcode's command line tools changed header layout.
```
Comment (bertmaher) [inline README.md]: grammar: Starting *with* macOS...

Comment (bertmaher) [reply README.md]: Maybe also add a comma, like "Starting with macOS Mojave, Xcode's command line..." but I'm nitpicking :)

Comment (UncleGene) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (UncleGene) [reply README.md]: Fixed

**Pull request closed without merging**